training_record/views.py

```diff
@@ -81,7 +81,7 @@ def reviewed(self, request):
                        url_path='department-admin-review')
     def department_admin_review(self, request, pk):
         '''Pass the record which is being reviewed.'''
-        is_approved = request.data.get('is_approved')
+        is_approved = request.data.get('approve')
```

Comment:
What will happen if `approve` is not presented in `request.data`, please prepare for that case.